packages/styled-system/src/css.ts

```diff
@@ -73,19 +90,22 @@ export function getCss(options: GetCSSOptions) {
         value = resolveTokenValue(theme, value)
       }
 
-      let config = configs[key]
+      if (isObject(value)) {
+        computedStyles[key] ??= {}
+        Object.assign(computedStyles[key], css(value, true))
+        continue
+      }
 
+      let config = configs[key]
       if (config === true) {
-        config = { property: key } as PropConfig
+        config = { property: key }
       }
-
-      if (isObject(value)) {
-        computedStyles[key] = computedStyles[key] ?? {}
-        computedStyles[key] = merge({}, computedStyles[key], css(value, true))
-        continue
+      if (!nested && config?.static) {
+        const staticStyles = runIfFn(config.static, theme)
+        Object.assign(computedStyles, staticStyles)
```

Comment:
- Immutable assignments are not needed, so I modified all assignments to be mutable
- Nested objects are processed recursively, therefore I don't think we need deep merging by `lodash.mergeWith`. So I fixed to use `Object.assign` instead of `lodash.mergeWith`